packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -93,6 +134,67 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 					} }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							displayAsRange: true,
+						} );
+					} }
+					dropdownMenuProps={ dropdownMenuProps }
+				>
+					<ToolsPanelItem
+						isShownByDefault
+						hasValue={ () =>
+							averageReadingSpeed !== AVERAGE_READING_RATE
+						}
+						label={ __( 'Average Reading Speed' ) }
+						onDeselect={ () =>
+							setAttributes( { averageReadingSpeed: undefined } )
```

Comment:
```suggestion
							setAttributes( { averageReadingSpeed: AVERAGE_READING_RATE } )
```

I think we need to update with the default value, otherwise resetting the setting won't work.